docket_scrap/spiders/docket_spider.py

```diff
@@ -0,0 +1,179 @@
+"""
+Scrapy Spider that crawls RIPUC webpage for commission dockets
+run by command "scrapy crawl docket" on cmd
+"""
+
+from w3lib.html import remove_tags
+import re
+import scrapy
+from urllib.parse import urljoin
+import csv
+
+from ..items import Docket
+
+# from scrapy.http.request import Request
+
+
+class DocketSpider(scrapy.Spider):
+    name = 'docket'
+    start_urls = [
+        'http://www.ripuc.ri.gov/eventsactions/docket.html'
+    ]
+    base_url = 'http://www.ripuc.ri.gov/eventsactions/'
+
+    def parse(self, response):
+        """
+            Handling the response of the webpage
+        """
+        rows = response.css("table")[2].css("table")[1].xpath('tr')
+
+        '''
+            temporary storage for dockets to check if there is any missing field
+        '''
+        docket_temporary_storage = dict()
+
+        for row in rows:
+            col = row.css('td')
+
+            if len(col) == 3:
+                '''
+                    when a row has three columns, it must have an ID, a Filer and a Description
+                '''
```

Comment:
Ok, to avoid confusion, when I said use docstrings, it means to add the relevant information in docstring. If needed, use inline comments where needed. Don't use docstring format for inline comments.